### ISSUE TITLE
Does it make sense to add bootstrap classes to the default template?

### ISSUE THREAD

**Issue by jankapunkt** [VeliovGroup/meteor-autoform-file#35]
I just realized, that autoform uses bootstrap, so this package could be extended by bootstrap class entries on the html level without hard-wiring bootstrap. I could provide an example PR if this can be considered an improvement.

**Comment by dr-dimitru** [comment]
Hello @jankapunkt ,

Sure, example for `uploadTemplate` which is used for `input[type="file"]`, and default is:
```html
      {{#with currentUpload}}
        Uploading <b>{{file.name}}</b>:
        <progress value="{{progress.get}}" max="100"></progress>
        &nbsp;
        <span class="progress">{{progress.get}}%</span>
      {{else}}
        <input data-files-collection-upload class="form-control af-file-upload-capture" type="file" />
      {{/with}}
```

`previewTemplate` is shown after file is uploaded, default:
```html
      <div>
        {{#if isImage}}
          {{> uploadImageDemo}}
        {{else}}
          {{> uploadFileDemo}}
        {{/if}}
      </div>
      <a data-reset-file href="#" style="display:none">Reset</a>
      <a data-remove-file href="#">Remove</a>
```

**Comment by jankapunkt** [comment]
implemented in PR #36

**Comment by dr-dimitru** [comment]
Closed due to silence at issue owner end.
Feel free to reopen it in case if the issue still persists on your end.
